src/crimson/os/seastore/random_block_manager/rbm_device.cc

```diff
@@ -236,5 +279,14 @@ write_ertr::future<> EphemeralRBMDevice::writev(
   return write_ertr::now();
 }
 
+EphemeralRBMDevice::mount_ret EphemeralRBMDevice::mount() {
+  return do_shard_mount();
+}
+
+EphemeralRBMDevice::mkfs_ret EphemeralRBMDevice::mkfs(device_config_t config) {
+  super.journal_size = DEFAULT_TEST_CBJOURNAL_SIZE;
+  return do_primary_mkfs(config, 1);
+}
```

Comment:
for EphemeralRBMDevice, only need support one shard, right?

Reply:
Right. That is why I added the number of shards as input argument, as explained above.